dist-persist/wbstack/src/Internal/ApiWbStackQueueSearchIndexBatches.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace WBStack\Internal;
+
+/**
+ * This should create the index for the requested wiki
+ */
+class ApiWbStackQueueSearchIndexBatches extends \ApiBase {
+    public function mustBePosted() {return true;}
+    public function isWriteMode() {return true;}
+    public function isInternal() {return true;}
+    public function execute() {
+        global $IP;
+
+        @set_time_limit( 60*5 ); // 5 mins maybe D:
+		@ini_set( 'memory_limit', '-1' ); // also try to disable the memory limit? Is this even a good idea?
+		
+        $parameters = "--queue --skipLinks --indexOnSkip --buildChunks 10000";
```

Comment:
If `--queue` is not needed here it might be worth removing as otherwise this is a little confusing.

With `--queue` it read that this is called, to get command to run from the API, but also that it itself queues things?